source/Octopus.Tentacle.Tests/Capabilities/CapabilitiesServiceFixture.cs

```diff
@@ -12,8 +12,7 @@ public void CapabilitiesAreReturned()
             new CapabilitiesService()
                 .GetCapabilities()
                 .SupportedCapabilities
-                .Count
-                .Should().Be(0);
+                .Should().BeEquivalentTo(new[]{ "IScriptServiceV2Alpha" });
```

Comment:
`IScriptServiceV2` is being added as `Alpha` to allow us to add the service and test it with branch instances of Server while not making it GA until we are happy with the functionality.